src/sentry/api/endpoints/organization_slugs.py

```diff
@@ -27,9 +29,9 @@ def put(self, request: Request, organization) -> Response:
         for project_id, slug in slugs.items():
             slug = slug.lower()
             try:
-                validate_slug(slug)
+                validate_sentry_slug(slug)
```

Comment:
Moved to `src/sentry/api/helpers/slugs.py`, ideally it would be in `src/sentry/api/utils.py` but this ran into a circular import